who_uses.md

```diff
@@ -4,14 +4,15 @@ title: Who Uses Foreman
 ---
 ## Who Uses Foreman?
 
-The following companies or individuals have confirmed use of Foreman. If you 
+The following companies or individuals have confirmed use of Foreman. If you
 want help to spread the word, [let us
 know](https://docs.google.com/a/zapletalovi.com/forms/d/1Ybz-Wz-odbGJM6Kw1pZuIOsZZ_vxXbH1I8B-iKFfuPs)
 and we will add you to the page.
```

Comment:
Not blocking this PR, but would it make sense to encourage a PR instead of using Google services that few have access to?

Reply:
@lzap ?

Reply:
I wanted to avoid this to keep privacy, we can do probably both.